packages/sanity/src/core/studio/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './activeWorkspaceMatcher'
 export * from './colorScheme'
+export * from './commentsSetup'
```

Comment:
I'd prefer having a `comments` root folder which has the `setup` folder, so that we can move other comments related things into the `comments` root folder in the future.